CHANGES.rst

```diff
@@ -18,6 +18,7 @@ Version 0.9.0 (2023-01-26)
 - Changes ``mv`` command to allow moving files while a workflow is running.
 - Changes ``upload`` command to prevent uploading symlinks.
 - Changes ``validation --environment`` command to use Docker registry v2 APIs to check that a Docker image exists in DockerHub.
+- Fixes ``du`` command help message typo.
```

Comment:
This line cannot go into the old release notes, because 0.9.0 has been already published... Please start a new section like:

```markdown
Version 0.9.1 (UNRELEASED)
--------------------------

- Fixes ``du`` command help message typo.
```

See the other repositories such as r-w-controller for example.

(And please squash the three commits together into one commit.)